<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watchdog configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="tracelayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Preconfigured conditions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="preconfTree">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="userWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>User-defined conditions:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="userTree">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="addBtn">
            <property name="toolTip">
             <string>Add a setpoint table for selected devices</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../resources/nicos-gui.qrc">
              <normaloff>:/add</normaloff>:/add</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="editBtn">
            <property name="toolTip">
             <string>Remove setting from the table</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../resources/nicos-gui.qrc">
              <normaloff>:/editor</normaloff>:/editor</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="delBtn">
            <property name="toolTip">
             <string>Remove setting from the table</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../resources/nicos-gui.qrc">
              <normaloff>:/remove</normaloff>:/remove</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
